statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -1335,6 +1335,226 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def test_normality(self, method):
+        """
+        Test for normality of standardized residuals.
+
+        Null hypothesis is normality.
+
+        Parameters
+        ----------
+        method : string {'jb'}
+            The statistical test for normality. Must be 'jarquebera' for
+            Jarque-Bera normality test.
+
+        Notes
+        -----
+        If the first `d` loglikelihood values were burned (i.e. in the
+        specified model, `loglikelihood_burn=d`), then this test is calculated
+        ignoring the first `d` residuals.
+
+        See Also
+        --------
+        statsmodels.stats.stattools.jarque_bera
+
+        """
+        if method == 'jarquebera':
+            from statsmodels.stats.stattools import jarque_bera
+            d = self.loglikelihood_burn
+            output = np.array(jarque_bera(
+                self.filter_results.standardized_forecasts_error[:, d:],
+                axis=1
+            )).transpose()
+        else:
+            raise NotImplementedError('Invalid normality test method.')
+
+        return output
+
+    def test_heteroskedasticity(self, method='', alternative='two-sided',
+                                asymptotic=False):
+        """
+        Test for heteroskedasticity of standardized residuals
+
+        Tests whether the sum-of-squares in the first third of the sample is
+        significantly different than the sum-of-squares in the last third
+        of the sample. Analogous to a Goldfeld-Quandt test.
+
+        Parameters
+        ----------
+        method : string {'sumsquares'}
```

Comment:
I'm not a fan of `sumsquares`, IMO sum of squares is an implementation detail. The main feature is that it tests a break in the variance. (like chow test)